tests/test_threadpoolctl.py

```diff
@@ -14,10 +15,10 @@
 from .utils import threadpool_info_from_subprocess
 
 
-def is_old_openblas(module):
+def is_old_openblas(libctl):
```

Comment:
Style: I think I would prefer the more explicit `lib_controller`.

Reply:
I was thinking the same but I thought it would make lines too long too often and hurt readability but actually it's fine